package.json

```diff
@@ -37,7 +37,8 @@
     "test:integration": "cross-env NODE_OPTIONS=--experimental-vm-modules jest test.ts --runInBand",
     "test:unit": "cross-env NODE_OPTIONS=--experimental-vm-modules jest spec.ts",
     "typecheck": "nuxi typecheck",
-    "validate": "yarn graphql:validate"
+    "validate": "yarn graphql:validate",
+    "generate-api": "graphql-codegen --config codegen.yml"
```

Comment:
Also shouldn't be necessary after merging the configs (done via the graphql:generate command defined above)